src/python/pants/base/build_environment.py

```diff
@@ -36,17 +36,6 @@ def get_pants_cachedir() -> str:
     return os.path.expanduser(os.path.join(cache_home, "pants"))
 
 
-def get_pants_configdir() -> str:
```

Comment:
This was only used to provide the default value for a now unused and deprecated global option.

Reply:
Thank you! Can also delete the rust `default_config_path` method as well probably.